smack-extensions/src/main/java/org/jivesoftware/smackx/muc/MultiUserChatManager.java

```diff
@@ -282,6 +282,9 @@ public boolean isServiceEnabled(Jid user) throws NoResponseException, XMPPErrorE
      * Returns a Set of the rooms where the user has joined. The Iterator will contain Strings where each String
      * represents a room (e.g. room@muc.jabber.org).
      *
+     * Note: A more modern way to determine which MUCs the user might be joined in on other clients is using
```

Comment:
I also believe this should go into the javadoc of `getJoinedRooms(EntityJid)` instead, no?

Reply:
As far as I understand, the bookmark manager is only used for the user themselves and cannot be used to determine bookmarks of contacts. `getJoinedRooms(EntityJid)` however can be used to determine, which mucs contacts are joined (right?). Therefore I decided to add the note only to the method that is used to query yourself.

Reply:
Right, I was confused.